admin/src/components/Tables/StatisticTable.vue

```diff
@@ -0,0 +1,91 @@
+<!-- eslint-disable vue/no-static-inline-styles -->
+<template>
+  <div class="statistic-table">
+    <b-table-simple
+      style="width: auto"
+      class="mt-5"
+      :items="items"
+      :fields="fields"
+      striped
+      stacked="md"
+    >
+      <b-thead>
+        <b-tr>
+          <b-th></b-th>
+          <b-th class="text-right">{{ $t('statistic.count') }}</b-th>
+          <b-th class="text-right">{{ $t('statistic.detail') }}</b-th>
+        </b-tr>
+      </b-thead>
+      <b-tbody>
+        <b-tr>
+          <b-td>
+            <b>{{ $t('statistic.totalUsers') }}</b>
+          </b-td>
+          <b-td class="text-right">{{ value.totalUsers }}</b-td>
+          <b-td></b-td>
+        </b-tr>
+        <b-tr>
+          <b-td>
+            <b>{{ $t('statistic.activeUsers') }}</b>
+          </b-td>
+          <b-td class="text-right">{{ value.activeUsers }}</b-td>
+          <b-td></b-td>
+        </b-tr>
+        <b-tr>
+          <b-td>
+            <b>{{ $t('statistic.deletedUsers') }}</b>
+          </b-td>
+          <b-td class="text-right">{{ value.deletedUsers }}</b-td>
+          <b-td></b-td>
+        </b-tr>
+        <b-tr>
+          <b-td>
+            <b>{{ $t('statistic.totalGradidoCreated') }}</b>
+          </b-td>
+          <b-td class="text-right">
+            {{ $n(value.totalGradidoCreated, 'decimal') }} {{ $t('GDD') }}
+          </b-td>
+          <b-td class="text-right">{{ value.totalGradidoCreated }}</b-td>
+        </b-tr>
+        <b-tr>
+          <b-td>
+            <b>{{ $t('statistic.totalGradidoDecayed') }}</b>
+          </b-td>
+          <b-td class="text-right">
+            {{ $n(value.totalGradidoDecayed, 'decimal') }} {{ $t('GDD') }}
```

Comment:
cant we use the GDD component here?

Reply:
The GDD component is not available in the admin area.